VagrantProvision.sh

```diff
@@ -7,14 +7,17 @@ source ~/.profile
 
 #To get rid of "dpkg-preconfigure: unable to re-open stdin: No such file or directory" warnings
 export DEBIAN_FRONTEND=noninteractive
+if [ -f /etc/apt/apt.conf.d/70debconf ]; then
+  sudo sed -i 's/ --apt//' /etc/apt/apt.conf.d/70debconf
+fi
```

Comment:
This removes the `dpkg-preconfigure` warning.  Just removes the `--apt` option that makes it not use `stdin` to check for options.